src/main/java/frc/robot/subsystems/pivot/PivotIOSim.kt

```diff
@@ -70,14 +71,19 @@ class PivotIOSim(
         sim.update(0.02)
     }
 
+    override fun stop() {
+        isPID = false
+        sim.setInputVoltage(0.0)
+    }
+
     /**
      * Sets the raw voltage applied to the motor(s).
      * @param voltage The voltage to apply.
      * @param isPID Whether the voltage is being set by a PID controller. (Used for simulation purposes.)
      */
     override fun setRawVoltage(voltage: Measure<Voltage>, isPID: Boolean) {
         cachedVoltage.mut_replace(voltage)
-        sim.setInputVoltage(voltage into Volts)
+        sim.setInputVoltage(-voltage into Volts)
```

Comment:
Wait, why negative?

Reply:
Just because sim motors don't have inversions, and neos and dcmotorsim seem to have different positive directions